ArduCopter/Parameters.h

```diff
@@ -381,12 +381,14 @@ class Parameters {
         k_param_vehicle = 257, // vehicle common block of parameters
         k_param_throw_altitude_min,
         k_param_throw_altitude_max,
-
+        k_param_land_accel_limit,       // 260
         // the k_param_* space is 9-bits in size
         // 511: reserved
+
```

Comment:
there are a couple of unnecessary white space changes.. not a big deal but I'd remove them if they're not intentional.

Reply:
Sure they are there by mistake.